drupal/web/themes/custom/genlyd_aarhus/templates/form/form--node-activity-form.html.twig

```diff
@@ -0,0 +1,186 @@
+{#
+/**
+ * @file
+ * Theme override for a 'form' element.
+ *
+ * Available variables
+ * - attributes: A list of HTML attributes for the wrapper element.
+ * - children: The child elements of the form.
+ *
+ * @see template_preprocess_form()
+ */
+#}
+
+{# TODO: step form #}
+{#
+{% set groups = form['#fieldgroups'] %}
+{% set group_keys = groups | keys %}
+#}
+      {#
+            {% embed "@layouts/grid-item.twig" %}
+              {% block content %}
+                {% set progressBar = { items: [
+                  {title: 'fisk'},
+                  {title: 'fusk'},
+                  {title: 'just'}
+                ] } %}
+                {% include '@modules/progress-bar.twig' %}
+              {% endblock %}
+            {% endembed %}
+      #}
+
+
+{% set form = element %}
+
+<form{{ attributes }}>
```

Comment:
Add as component in Pattern Lab and embed so we avoid maintaining multiple components.